packages/aws-cdk/lib/cli.ts

```diff
@@ -604,6 +606,7 @@ export async function exec(args: string[], synthesizer?: Synthesizer): Promise<n
           assetParallelism: configuration.settings.get(['assetParallelism']),
           assetBuildTime: configuration.settings.get(['assetPrebuild']) ? AssetBuildTime.ALL_BEFORE_DEPLOY : AssetBuildTime.JUST_IN_TIME,
           ignoreNoStacks: args.ignoreNoStacks,
+          exitOnConfigComplete: args.exitOnConfigComplete,
```

Comment:
on line 658 of this file, for `case 'destroy'`. do we also need to pass this exitOnConfigComplete? What about `case: 'watch'`?

Reply:
Just confirmed the optimistic strategy from CFN does not support deleting stack so I will remove that from `destroy`. We'll only support `deploy`.